lib/twirp/client.rb

```diff
@@ -96,8 +96,13 @@ def is_http_redirect?(status)
         status >= 300 && status <= 399
       end
 
-      def rpc_path(service_full_name, rpc_method)
-        "/#{service_full_name}/#{rpc_method}"
+      def make_http_request(conn, service_full_name, rpc_method, content_type, body)
+        conn.post do |r|
+          r.url "#{service_full_name}/#{rpc_method}"
```

Comment:
maybe use the class-level method `Client#rpc_name` here? Since it removes the usage of magic strings across the codebase.